DTS-STN/Alpha-Site
# ISR again

# Description

[ISR](https://trello.com/c/AHRiZFGP)

After some trial and error, ISR should work now. Testing locally was misleading because there were no problems, but it didn't work in the docker container. After testing, I found that the envs we used for the Strapi api and the ISR feature flag needed to be made public in order for them to work as expected.

## Acceptance Criteria

ISR actually works.

## Test Instructions

Since this works as is when running `npm run dev`, the test instructions are to build the docker container and run that as that's where the issue was.

1. Have docker installed
2. Run `docker build -t <name of the image> --build-arg NEXT_PUBLIC_NOTIFY_REPORT_A_PROBLEM_EMAIL=<an email you have access to> --build-arg NEXT_BUBLIC_STRAPI_API_BACKEND_URL=https://alphasite-api.dts-stn.com --build-arg NEXT_PUBLIC_ISR_ENABLED=true .`
3. After the image is built, run `docker run -p 3000:3000 <name of the image you used above>`
4. Go to `localhost:3000/projects` and see the projects
5. In a separate tab, go to `https://alphasite-admin.dts-stn.com/`
6. Edit, create, or remove (or unpublish) an experiment and save
7. Go back to the projects page and refresh. It could take a minute before you see the changes, so refresh a few times

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
